Issue1437 1614 ets indirect cooling

khinkelman opened the pull request:
@JayHuLBL, I discovered performance errors in these models that have now been corrected. In the two example models, the control valves were opening 100% for the entire simulation primarily due to incorrectly set reference pressures & drops. Please let me know if you have any questions/comments.

JayHuLBL commented:
@khinkelman 
Thanks for reporting. I will review it on weekend.

JayHuLBL commented:
@khinkelman 
- [ ] In `Buildings.Applications.DHC.EnergyTransferStations.CoolingIndirect`, change following:
```
  Modelica.Blocks.Continuous.Integrator int(k=1) "Integration"
    annotation (Placement(transformation(extent={{60,120},{80,100}})));
```
to 
```
  Modelica.Blocks.Continuous.Integrator int(k=1) "Integration"
    annotation (Placement(transformation(extent={{60,100},{80,120}})));
```
- [ ] In both `Buildings.Applications.DHC.EnergyTransferStations.Examples.CoolingIndirectClosedBuildingLoop` and `Buildings.Applications.DHC.EnergyTransferStations.Examples.CoolingIndirectOpenLoops`, add comment to instance 
```
package Medium = Buildings.Media.Water;
```

Also, since the CI test has failed, would you please do following (in this PR and PR #1690):
- [ ] merge the `lbl-srg:issue1437_1614_ets_indirectCooling` to your branch.
- [ ] run unit test and update the reference results.

JayHuLBL commented on Buildings/Applications/DHC/EnergyTransferStations/CoolingIndirect.mo:
```diff
@@ -1,9 +1,10 @@
     redeclare final package Medium1 = Medium,
-    redeclare final package Medium2 = Medium);
+    redeclare final package Medium2 = Medium,
+    show_T = true);
```
add
`final m1_flow_nominal = mDis_flow_nominal,
  final m2_flow_nomnal = mBui_flow_nominal,
`
and change the name of below parameter `m1_flow_nominal` to `mDis_flow_nominal`, `m2_flow_nominal` to `mBui_flow_nominal`.
The reason for the change is that current setting would trigger warning since the attribute `start = 0.5` is not in the `m1_flow_nominal` of class `Buildings.Fluid.Interfaces.PartialFourPortInterface`.

JayHuLBL commented on Buildings/Applications/DHC/EnergyTransferStations/CoolingIndirect.mo:
```diff
@@ -207,7 +208,7 @@ model CoolingIndirect
-  Buildings.Fluid.Actuators.Valves.TwoWayQuickOpening val(
+  Buildings.Fluid.Actuators.Valves.TwoWayEqualPercentage val(
     redeclare final package Medium = Medium,
     final m_flow_nominal=m1_flow_nominal,
```
change it to `final m_flow_nominal=mDis_flow_nominal,`

khinkelman commented:
Thanks @JayHuLBL, I merged `lbl-srg:issue1437_1614_ets_indirectCooling` to my branch, but my branch was already up to date. The CI tests are failing primarily because the docker login is not receiving an input. I expect there is some adjustment you need to make on your end? If there is anything else I need to try or do differently, please let me know. I'd be happy to discuss over the phone as well if that helps.

JayHuLBL commented:
After you are done with other changes, I can merge it to lbl-srg repository to see if the CI test will still be failing.

khinkelman commented:
Sounds good. I'll let you know when I am finished with the changes.

khinkelman commented:
Hi @JayHuLBL, I finished addressing your comments, merged LBL's indirect/direct cooling branches to my forked branches, and reran unit tests for my new models. The CI builds are in progress. (One question: should I be running unit test for the entire library or just my new packages?) Please let me know if there is anything else you need me to do. Thanks!

JayHuLBL commented:
@khinkelman Regarding the unit test, you don't need to run the entire library. Just run the package that have your changs.
